### DISCUSSION TITLE
X/Y Min Position

### DISCUSSION THREAD

**Discussion by cjourdan87** [mriscoc/Ender3V2S1#927] [General]
I am having some troubles setting up my min/max positions for my bed.  I am using an Ender 3 S1 Pro, and was using the default settings until I started printing larger things and realized it was printing off centered.

Let me first say that I know there is a guide, and I have gone through it a few times, but I seem to be missing something.
I auto home and set min positions to 0 and find my (0,0) is inside of the bed.  This is where I believe I am messing up.  

The guide then says to move the nozzle until it is above the front left corner of the bed, but I cannot move it because it won't go lower than the (0,0), so I adjust it to a lower negative value.  The numbers I came up with for the nozzle above the corner this last time was (-15,-8), so I put those in as the minimum positions respectively under Advanced>Physical Settings>X min and Y min.

I save, auto home, and then do the (0,0). It says the nozzle should now be over the corner of the bed, but it is still inside of the bed, and it is reading (0,0) on the screen.

What am I missing or doing wrong?  Am I just not understanding the whole procedure?

**Comment by mriscoc** [comment]
Did you save settings and reboot?

**Reply by cjourdan87** [reply]
I did save, but I do not recall if I rebooted.  I will give that a try and give an update.  I do appreciate the response.  Thank you.

**Reply by cjourdan87** [reply]
The rebooting does seem to have done the trick!  Thank you for the feedback and keep up the awesome work!  The firmware is really good!
